src/java.desktop/share/classes/sun/awt/SunToolkit.java

```diff
@@ -1465,19 +1465,20 @@ public void realSync() {
      * ((SunToolkit)Toolkit.getDefaultToolkit()).realSync();
      * }</pre>
      *
-     * <p> After realSync, {@code b} will be focus owner.
+     * <p> After {@code realSync}, {@code b} will be focus owner.
      *
-     * <p> Notice that realSync isn't guaranteed to work if recurring
-     * actions occur, such as if during processing of some event
+     * <p> Notice that {@code realSync} isn't guaranteed to work if recurring
+     * actions occur, such as if during processing of an event
      * another request which may generate some events occurs.  By
-     * default, sync tries to perform as much as {@value #MAX_ITERS}
+     * default, sync tries to perform as many as {@value #MAX_ITERS}
      * cycles of event processing, allowing for roughly {@value
      * #MAX_ITERS} additional requests.
      *
-     * <p> For example, requestFocus() generates native request, which
-     * generates one or two Java focus events, which then generate a
-     * serie of paint events, a serie of Java focus events, which then
-     * generate a serie of paint events which then are processed -
+     * <p> For example, {@link Component#requestFocus() requestFocus()}
+     * generates a native request, which generates one or two Java
+     * focus events, which then generate a series of paint events,
+     * a series of Java focus events, which then generate
+     * a series of paint events which then are processed &mdash;
```

Comment:
The only change here is I added the link to `requestFocus` and replaced hyphen with mdash. I re-flowed the text for easier reading in the source code.

Probably, this text could be improved…